docs/components/datetimepicker.md

```diff
@@ -0,0 +1,409 @@
+        { key: "previous_month", prefix: "Previous", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+  ]
+/>
+```
+
+### Relative time frames only
+
+Set the `v-model` to [Range date time picker](#range-date-time-picker-v-model)
+Same time frames as the previous example, except now we're passing in a pre-selected time frame.
+
+<div>
+  <KDateTimePicker
+    v-model="currentValue6"
+    @change="newVal => emitVal6 = newVal"
+    placeholder="Please select a time frame"
+    mode="relative"
+    :timePeriods="exampleTimeFrames"
+  />
+  <div class="mt-6">Emitted value: <pre class="json">{{ emitVal6 }}</pre></div>
+</div>
+
+```html
+<KDateTimePicker
+  v-model="currentValue"
+  @change="newVal => emitVal = newVal"
+  placeholder="Please select a time frame"
+  mode="relative"
+  :timePeriods=[
+    {
+      section: "Last",
+      values: [
+        { key: "15m", prefix: "Last", timeframeText: "15 minutes", start: function() {}, end: function() {} },
+        { key: "12h", prefix: "Last", timeframeText: "12 hours", start: function() {}, end: function() {} },
+        { key: "24h", prefix: "Last", timeframeText: "24 hours", start: function() {}, end: function() {} },
+        ...
+      ]
+    },
+    {
+      section: "Current",
+      values: [
+        { key: "current_week", prefix: "Current", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "current_month", prefix: "Current", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+    {
+      section: "Previous",
+      values: [
+        { key: "previous_week", prefix: "Previous", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "previous_month", prefix: "Previous", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+  ]
+/>
+```
+
+## Props
+
+### v-model
+
+#### Single date time picker `v-model`
+
+A single date or time picker instance will emit a Javascript Date object as a `string`.
+
+```js
+"2022-09-06T19:45:54.406Z"  // ISO 8601 Date string
+```
+
+#### Range date time picker `v-model`
+
+An instance that displays a date range or a series of relative time frames will emit an `object` containing the following values:
+
+```js
+{
+  "start": "2022-09-06T19:45:54.406Z",  // ISO 8601 Date string
+  "end": "2022-09-07T19:45:54.406Z", // ISO 8601 Date string
+  "timePeriodsKey": "24h" // or "" if not applicable
+}
+```
+
+### maxDate
+
+`Number` which blocks out days/times that occur **after** the given timestamp. `maxDate` gets passed down to the calendar component, and does not apply to relative time frames.
+
+**default**: `null`
+
+### minDate
+
+`Number` which blocks out days/times that occur **before** the given timestamp. `minDate` gets passed down to the calendar component, and does not apply to relative time frames.
+
+**default**: `null`
+
```

Comment:
1. issue: The props are actually typed to `Date`, meaning instead of telling the user to pass in a `Number` we should be telling them here to pass in a valid `Date` object. Can we update the descriptions (and possibly add an example) of what the user can actually pass in? 
2. nit: Can you switch the order of these props so that `minDate` is first, and `maxDate` is second?